pkg/instance/sidecars.go

```diff
@@ -92,12 +94,16 @@ func (s *sidecars) setStateForSidecars(state InstanceState) {
 		})
 }
 
-func (s *sidecars) cloneWithSuffix(suffix string) *sidecars {
+func (s *sidecars) clone(namePrefix string) (*sidecars, error) {
```

Comment:
Why the change from suffix to prefix?
Doesn't suffix make it easier to see everything that is related? printed out alphabetically?

Reply:
When clone an instance, we need to clone also the sidecars and to avoid name collision, we use the instance name as the prefix for its sidecar's names.